src/coreclr/jit/lsraxarch.cpp

```diff
@@ -1297,7 +1297,7 @@ int LinearScan::BuildBlockStore(GenTreeBlk* blkNode)
         switch (blkNode->gtBlkOpKind)
         {
             case GenTreeBlk::BlkOpKindUnroll:
-                if (size >= XMM_REGSIZE_BYTES)
+                if (AMD64_ONLY(!blkNode->MustUsePointerSizeAtomicStores() &&)(size >= XMM_REGSIZE_BYTES))
```

Comment:
I have not seen these defines in the Jit and I would prefer not to.

Reply:
Sure I will change this